infra/cifuzz/actions/action.yml

```diff
@@ -0,0 +1,12 @@
+# action.yml
+name: 'build-fuzzers'
+description: "Builds an OSS-Fuzz project's fuzzers."
+inputs:
+  oss-fuzz-project-name:
```

Comment:
nit: is the "oss-fuzz" prefix necessary here? Same for the env var. Seems like we can just drop it given the context.

Reply:
It is necessary for the user to pass in the OSS-Fuzz project name just incase it is different than the repo name. For example nDPI's OSS-Fuzz project name is ndpi but repo name is nDPI.

Reply:
I still don't understand if the prefix is needed.

Reply:
I miss understood the context of the comment. I will make the required change.